nengo_loihi/tests/test_learning.py

```diff
@@ -64,6 +66,66 @@ def test_pes_comm_channel(allclose, plt, seed, Simulator, n_per_dim, dims):
     assert allclose(y_loihi, y_nengo, atol=0.15, rtol=0.1)
 
 
+def test_pes_overflow(allclose, plt, seed, Simulator):
+    dims = 1
```

Comment:
Would be good to test with `dims > 1` (so that we're checking this works correctly for multiple dimensions).

Reply:
done